src/constants/routes.js

```diff
@@ -1,4 +1,8 @@
-export const HOME_ROUTE = '/'
-export const AUTH_ROUTE = '/auth'
-export const SIGN_IN_ROUTE = '/auth/login'
-export const SIGN_UP_ROUTE = '/auth/signup'
+export const HOME_ROUTE = '/';
+export const AUTH_ROUTE = '/auth';
+export const SIGN_IN_ROUTE = '/auth/login';
+export const SIGN_UP_ROUTE = '/auth/signup';
+export const MY_PROJECTS_ROUTE = '/projects/:slug';
+export const PROJECT_REPORT_ROUTE = '/projects/:slug/:projectSlug';
```

Comment:
why you use double slug ? it's easier if you use a prefix route like admin/ or report/, something referring to the report.